### ISSUE TITLE
Fix infinite scroll issue while loading inscriptions

### ISSUE THREAD

**Issue by markmhendrickson** [blockstack/blockstack-app#3783]
E.g. with bc1p726hjpf35r2tmz58d9m6ttp44gpvczla4tmumshs773n65hxeucs0caszz

https://github.com/hirosystems/wallet/assets/28991/6032bbf0-78fe-4292-bcff-2323d7985b3f

**Comment by 314159265359879** [comment]
This looks related to https://github.com/hirosystems/token-metadata-api/issues/158#issuecomment-1556029594
I have seen Stacks NFT's (only two in the wallet) displayed 10-20 times in the wallet. This bug has existed for a longer time (not related to infinite loading). 

I have also noticed that the order of inscriptions seems to be changing. After infinite loading I see BRC are below Stacks NFT's prior to this they were displayed above. 

When I tested the infinite loading PR I did not see this. I wonder if it is related to Stacks NFT's loading above. I would expect the inscriptions to load below. 


https://github.com/hirosystems/wallet/assets/33360391/f82938bd-b483-49ec-b976-b7be323a7fee


There is no scrolling bar visible, is that intended?
